src/demo/app/examples/forms-async-data-example/forms-async-data-example.component.ts

```diff
@@ -1,5 +1,5 @@
 import { Component, OnInit } from '@angular/core';
-import { FormBuilder, FormGroup } from '@angular/forms';
```

Comment:
If you revert the `Untyped*` changes in all of the demo apps does this still build? If it does you might as well opt into the type checking now and not include these updates.

Reply:
This is an automatic migration by CLI. In v14 all `FormGroup`, `FormControl` and `FormBuilder` should have types. If not they need to be converted to `UntypedFormGroup`, `UntypedFormControl` and `UntypedFormBuilder`

Reply:
Yes, I understand this is a migration, but if you revert these changes and it still builds then the types are all valid and used correctly. Angular added the migration so that if the build broke you wouldn't have to convert everything to `Untyped*` manually. I was just suggesting reverting all the changes made by the migration and seeing if the migration wasn't needed. In a project I am working on I did the same and only two components had typing issues and it was easy enough to change them now.